src/pages/OnboardingPurpose/BaseOnboardingPurpose.tsx

```diff
@@ -83,6 +91,11 @@ function BaseOnboardingPurpose({shouldUseNativeStyles, shouldEnableMaxHeight, ro
                 Welcome.setOnboardingErrorMessage('');
 
                 if (choice === CONST.ONBOARDING_CHOICES.MANAGE_TEAM) {
+                    if (!onboardingPolicyID && !hasSignupQualifier && filteredPolicies.length === 0) {
+                        const {adminsChatReportID, policyID} = Policy.createWorkspace(undefined, true, '', generatePolicyID(), choice);
+                        Welcome.setOnboardingAdminsChatReportID(adminsChatReportID);
+                        Welcome.setOnboardingPolicyID(policyID);
+                    }
```

Comment:
I'm confused about why we moved this here.

Reply:
We moved the workspace creation logic here to `BaseOnboardingPurpose` because this page is only shown to users without VSB or SMB signup qualifiers. For users with VSB or SMB, this page is skipped, and the workspace creation won’t happen on the frontend.

I believe the workspace creation was temporarily moved to `BaseOnboardingEmployees` in your PR (https://github.com/Expensify/App/pull/50759) until the backend changes are ready. Now that we’ve moved the logic to BaseOnboardingPurpose, the workspace will only be created for users without VSB or SMB signup qualifiers, which is what we need to avoid duplicates.

Reply:
We have to fix this now because the FE needs will be merged first. Can you please return it to the correct spot?